.github/workflows/pr-title-check.yaml

```
@@ -43,6 +43,13 @@ jobs:
           echo "Description: $PR_DESCRIPTION"
           
           # Parse JSON from validation.yaml
+          # Confirm the keyword exists.
+          keyword_exists="$(
+            yq e ".allowed_keywords[] | select(.name == \"$COMMIT_KEYWORD\") | .name" \
+            "$validation_yaml"
+          )"
+          echo "Is keyword $Keyword exists in allowed_keywords?: $keyword_exists"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)